src/services/profiles/logs.ts

```diff
@@ -0,0 +1,113 @@
+import type { HttpInstance } from '../../types';
+import {
+  AffectedRecords,
+  ObjectId,
+  PagedResult,
+  Results,
+  ResultResponse,
+} from '../types';
+import { LogEntry, ProfileComment } from './types';
+
+export default (client, httpAuth: HttpInstance) => ({
+  /**
+   * Create a profile log entry
+   * Permission | Scope | Effect
+   * - | - | -
+   * `CREATE_PROFILE_LOG_ENTRIES` | `staff enlistment` | Create a log entry for any profile of this group
+   * `CREATE_PROFILE_LOG_ENTRIES` | `global` | Create a log entry for any profile of any group
+   *
+   * @param profileId Id of the targeted profile
+   * @param groupId Id of the targeted group
+   * @param requestBody ProfileComment
+   * @returns LogEntry
+   * @throws {ResourceUnknownError}
+   */
+  async create(
+    profileId: ObjectId,
+    groupId: ObjectId,
+    requestBody: ProfileComment
+  ): Promise<LogEntry> {
+    return (
+      await client.post(
+        httpAuth,
+        `/${profileId}/groups/${groupId}/logs`,
+        requestBody
+      )
+    ).data;
+  },
+
+  /**
+   * Retrieve all profile log entries
+   * Permission | Scope | Effect
+   * - | - | -
+   * `VIEW_PROFILE_LOG_ENTRIES` | `staff enlistment` | Retrieve a list of log entries for any profile of this group
+   * `VIEW_PROFILE_LOG_ENTRIES` | `global` | Retrieve a list of log entries for any profile of any group
+   *
+   * @param profileId Id of the targeted profile
+   * @param groupId Id of the targeted group
+   * @returns PagedResult<LogEntry>
+   * @throws {ResourceUnknownError}
+   */
+  async getLogs(
```

Comment:
Hmm now it's `sdk.profiles.logs.getLogs` 
But `find` also sounds misleading as you can not pass in any filter.
Maybe `getAll` ? or `list` ?

Reply:
On a closer look, it does seem as you can pass in RQL. The result is also paged, so you would need at least be able to pass in `limit`. So just call it `find` ?

Reply:
> Hmm now it's `sdk.profiles.logs.getLogs`
> But `find` also sounds misleading as you can not pass in any filter.
> Maybe `getAll` ? or `list` ?

agree, see my comment below

Reply:
> On a closer look, it does seem as you can pass in RQL. The result is also paged, so you would need at least be able to pass in `limit`. So just call it `find` ?

Where do you see the RQL?